app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss

```diff
@@ -887,7 +889,7 @@ html.omni-nav-v2 {
       padding: 1px 0 0 20px;
       border-left: 1px solid #2f2f2f;
       font-size: 18px;
-      font-family: Minion Pro;
+      font-family: 'minion-pro';
```

Comment:
@waidelacruz Shouldn't there be some fallback fonts here?

    font-family: minion-pro, serif;

Reply:
If serif is inherited somewhere else, then my question can be disregarded.